tests/test_unique.py

```diff
@@ -0,0 +1,62 @@
+import pytest
+
+from faker import Faker
+from faker.exceptions import UniquenessException
+
+
+class TestUniquenessClass:
```

Comment:
I would add some tests to show that using the `unique` proxy will not interfere with the default behavior like the `_dummy = [faker.boolean() for _ in range(num_samples)]` thing I mentioned in another comment.